Project.toml

```
@@ -39,6 +42,9 @@ TreeViews = "0.3"
 UnPack = "1.0.2"
 julia = "1.6"
 
+[extensions]
+FastExtendedJumpArrayBroadcast = "FastBroadcast"
```

```suggestion
JumpProcessFastBroadcastExt = "FastBroadcast"
```

The naming for the module should follow the standard format.